tests/testthat/test-with.R

```diff
@@ -18,7 +18,7 @@ test_that("all default linters are tagged default", {
   # covr modifies package functions causing differing deparse() results even for identical anonymous functions.
   # This happens because default_linters is generated at build time and thus not modifiable by covr, whereas
   # linters_with_tags() constructs the linters at runtime.
-  skip_if(requireNamespace("covr", quietly = TRUE) && covr::in_covr())
+  skip_on_covr()
 
   expect_equal(linters_with_tags("default"), linters_with_defaults())
```

Comment:
@IndrajeetPatil oh, whoops, we should remove `expect_identical_linter()` since we didn't apply that internally...

Reply:
#1561